pkg/log/hclog.go

```diff
@@ -0,0 +1,58 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/hashicorp/go-hclog"
+	"google.golang.org/grpc/metadata"
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	"kusionstack.io/kusion-module-framework/pkg/util/kfile"
+)
+
+const (
+	KusionModuleName = "KUSION_MODULE_NAME"
```

Comment:
It seems that keys in the context are usually transferred into lowercase as referenced in [gRPC metadata](https://github.com/grpc/grpc-go/blob/4c07bca27377feb808912b844b3fa95ad10f946b/metadata/metadata.go#L59). You can also consider use `kusion_module_name` and `kusion_trace_id` here.

Reply:
fixed